tests/regression/tests/REQUEST-934-APPLICATION-ATTACK-GENERIC/934120.yaml

```diff
@@ -694,12 +694,12 @@ tests:
             port: 80
             headers:
               Host: "localhost"
-              User-Agent: "ModSecurity CRS 3 Tests"
+              User-Agent: "OWASP CRS test agent"
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
               Accept-Encoding: gzip,deflate
               Accept-Language: en-us,en;q=0.5
             method: GET
-            uri: '/get?ssrf=http://\\l\\o\\c\\a\\l\\h\\o\\s\\t/'
+            uri: "/get?ssrf=http://\\\\l\\\\o\\\\c\\\\a\\\\l\\\\h\\\\o\\\\s\\\\t/"
```

Comment:
I'm unsure about these changes (the other is at 934120-42). Is it really better to move from single quotes to double quotes and add all these escapes?

Reply:
I wanted to union how `uri` looks, including quotes.

Reply:
I understand, but it makes reading and understanding a lot harder. If not necessary, I'd try to avoid double quotes, at least for complex strings. They are necessary if you want to introduce non-printable characters.